server/index.js

```diff
@@ -147,13 +147,15 @@ app.get("/*", async (req, res) => {
     }
   }
 
+  /*
```

Comment:
In all fairness, it does say "[Do not merge]" on the PR :)
But when time allows, perhaps you can explain this.

Reply:
@peterbe This would catch the fundamental redirects which do not start with `/docs/`. 

The comment refers to
| All *documents* should be handled by the `if (req.url.includes("/docs/"))` test above.
But that line doesn't exist (anymore).

Reply:
OK. So by commenting it out, it means that your new code gets a chance to breath deeper in this Express request handler. I.e. around line ~211. 

Perhaps, we should implement our server handler a bit more similar to how Lambda@Edge will work in that it will scrutinize the URL (with the regexes) *before* it even bothers to do any lookups. 
For example, *above* the line `  if (!req.url.includes("/docs/")) {` we put in something like:
```javascript
const fundamentalRedirect = resolveFundamental(url);
if (fundamentalRedirect) {
  return redirect(301, fundamentalRedirect)
}
```
Does that make sense?

Reply:
makes sense